### ISSUE TITLE
`dataset_dict` is static and doesn't track changes in the `DATA_PATH`

### ISSUE THREAD

**Issue by VPetukhov** [euxhenh/cellar#11]
At the moment, uploading a new dataset to the `DATA_PATH` through command line (not through cellar UI) doesn't change the drop-down content in the [dataset selector](https://github.com/euxhenh/cellar/blob/a1b9e551ed1ba29a7808df9dc9cc4359aae27008/layout/navbar.py#L217). So, every time I upload a dataset to the server, cellar needs to be restarted. One way to resolve it would be to update the drop-down content on every click to [Load Data](https://github.com/euxhenh/cellar/blob/a1b9e551ed1ba29a7808df9dc9cc4359aae27008/layout/navbar.py#L45). Do you think it's a reasonable solution? I could make a PR to fix it.

**Comment by euxhenh** [comment]
I am not sure I understand your point. Do you mean the dropdown doesn't update when you upload through the interface, or when you `mv` a dataset into the data folder? If it is the former, then this sounds like a bug.

**Comment by VPetukhov** [comment]
The latter one. I expect a constant inflow of datasets, which I want all users to see. So I'll be copying new `h5ad` objects to the `data/server` folder a lot. But currently I'd need to restart the server to make the users see these changes.

**Comment by euxhenh** [comment]
I see. In that case we could add a "refresh" button, or update the dataset list whenever the dropdown menu is clicked. I prefer the latter as it requires less user input. (Updating upon clicking Load Data sounds a bit less intuitive to me). Let me know what you think.

**Comment by VPetukhov** [comment]
Yep, I agree that refreshing on 'Load data' would be the best. Thanks for your response!
